grid_map_ros/test/GridMapRosTest.cpp

```diff
@@ -72,14 +77,20 @@ TEST(RosbagHandling, saveLoad)
 
   EXPECT_FALSE(gridMapOut.exists(layer));
 
+  // Cleaning incase the previos bag was not removed
```

Comment:
```suggestion
  // Cleaning in case the previous bag was not removed
```

Reply:
Looks like the spell checker plugin is doing a fine job :')